tests/serverpod_test_server/test/serialization_test.dart

```diff
@@ -259,52 +296,61 @@ void main() {
       expect(unpacked.aNullableListWithNullableByteDatas![0]!.lengthInBytes,
           equals(256));
       expect(unpacked.aNullableListWithNullableByteDatas![1], isNull);
+
+      expect(unpacked.aNullableDurationList!.length, equals(2));
+      expect(unpacked.aNullableDurationList![0].inSeconds, equals(1));
+      expect(unpacked.aNullableDurationList![1].inMinutes, equals(1));
+
+      expect(unpacked.aNullableListWithNullableDurations!.length, equals(2));
+      expect(unpacked.aNullableListWithNullableDurations![0]!.inSeconds,
+          equals(1));
+      expect(unpacked.aNullableListWithNullableDurations![1], isNull);
     });
 
     test('Map types', () {
-      var maps = ObjectWithMaps(dataMap: {
```

Comment:
Please keep the same formatting, so that only relevant changes to the new feature is being made. It makes it easier to review the pull request, and also makes it easier to work with the code going forward.

Reply:
I run `dart format .` in each package, i hope it should fix

Reply:
`dart format` is very sensitive to where you place the commas. In this case, you have added a comma at the end of the map which changes the formatting.